src/handlers/swap_handlers.ts

```diff
@@ -166,6 +177,15 @@ const parseGetSwapQuoteRequestParams = (req: express.Request): GetSwapQuoteReque
             ? undefined
             : parseStringArrForERC20BridgeSources(req.query.excludedSources.split(','));
     const affiliateAddress = req.query.affiliateAddress;
+    const rfqt =
+        req.query.intentOnFilling === undefined
+            ? undefined
+            : {
+                  intentOnFilling: ['true', ''].includes(req.query.intentOnFilling) ? true : false,
+              };
+    // tslint:disable-next-line:boolean-naming
+    const skipValidation =
+        req.query.skipValidation === undefined ? false : ['true', ''].includes(req.query.skipValidation) ? true : false;
```

Comment:
Why do we want to consider `req.query.skipValidation === ''` as true?

Reply:
Same logic behind my recent change to `intentOnFilling`.  I think it makes logical sense to be able to say just `/quote?skipValidation&...`, and when that's the query string, `req.query` here contains `"skipValidation": ""`.